docs/spec_draft.md

```diff
@@ -388,6 +389,50 @@ dimensions `k` in `operand`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.case
+
+### Semantics
+
+Returns the result of exactly 1 function from `branches` depending on the value
+of `index`. Formally, if $0 \le$ `index` $\lt$ `N-1`, result of executing
+`branches[index]` is returned, else, result of executing `branches[N-1]` is
+returned.
```

Comment:
Also, let's call out that only one branch is executed, e.g. "Only one branch is executed depending on the value of `index`". Related discussion: https://github.com/openxla/stablehlo/pull/325/files#r996108043.

Reply:
Produces the output from executing exactly one `function` from `branches`
depending on the value of `index`. Formally, if $0 \le$ `index` $\lt$ `N-1`,
output of `branches[index]` is returned, else, output of `branches[N-1]` is
returned.

Second sentence still has return, it seems a better fit for second sentence. First line captures exactly one branch being executed. Let me know your thoughts.